src/main/java/com/toofifty/easyblastfurnace/EasyBlastFurnacePlugin.java

```diff
@@ -84,6 +84,8 @@ public class EasyBlastFurnacePlugin extends Plugin
     @Getter
     private boolean isEnabled = false;
 
+    private int oreOntoConveyorCount = 0;
```

Comment:
Any way we could move this & its logic outside of the main class? It's a bit messy having it here, it should probably belong with the coal bag implementation.

It's usually best to separate concerns and hide implementation details as much as possible, especially with a main class like this. I like to make the event handlers as thin as possible, and pass off control to some other service straight away

So the event handler body could read something like this:
```java
if (emptyMatcher.matches()) {
    state.getCoalBag().empty();

} else if (filledMatcher.matches()) {
    state.getCoalBag().fill();

} else if (/* test message */) {
    state.getCoalBag().placeOnToConveyor();

}

Reply:
I've done as you asked for this in the new commit. Definitely a good idea to not clog up the plugin class with random stuff.